web/src/components/TestResults/TestResults.tsx

```diff
@@ -90,23 +47,12 @@ const TestResults = () => {
       {!isLoading && Boolean(assertionResults) && (
         <TestSpecs
           assertionResults={assertionResults!}
-          onDelete={handleDelete}
-          onEdit={handleEdit}
+          onDelete={onDelete}
```

Comment:
Should these functions be defined at the TesSpecs component level instead?

Reply:
We are also using those functions at a higher level. Probably they can be moved to the TestSpecs provider.